testsuite/features/step_definitions/command_steps.rb

```diff
@@ -66,7 +67,8 @@
     # TODO: move that internal resource to some other external location
     STDERR.puts 'Sanity check not implemented, move resource to external network first'
   else
-    url = 'https://registry.mgr.suse.de:443'
+    registry_uri = ENV['REGISTRY_URI']
```

Comment:
Same remark: if we make it non-mandatory in sumaform, then it's possible this variable is just empty. If that's the case, we just skip the tests implying a registry.

Reply:
resolved